Dockerfile.template

```diff
@@ -33,8 +33,8 @@ LABEL microcosm.postgres=true
 
 ARG BUILD_NUM
 ARG SHA1
-ENV MICROCOSM-POSTGRES__BUILD_INFO_CONVENTION__BUILD_NUM ${BUILD_NUM}
-ENV MICROCOSM-POSTGRES__BUILD_INFO_CONVENTION__SHA1 ${SHA1}
+ENV MICROCOSM_POSTGRES__BUILD_INFO_CONVENTION__BUILD_NUM ${BUILD_NUM}
+ENV MICROCOSM_POSTGRES__BUILD_INFO_CONVENTION__SHA1 ${SHA1}
 COPY $NAME /src/$NAME/
-RUN pip install --extra-index-url $EXTRA_INDEX_URL -e .
+RUN pip install --extra-index-url $EXTRA_INDEX_URL -e .[encryption]
```

Comment:
This isn't standard, but it's needed for testing some unit tests. It will be overridden as part of the next build gen unless globality-build is updated to accommodate.